keras_core/ops/core.py

```diff
@@ -293,6 +293,54 @@ def stop_gradient(variable):
     """
     return backend.core.stop_gradient(variable)
 
+class ForiLoop(Operation):
+    def __init__(self, lower, upper, body_fun):
+        super().__init__()
+        self.lower = lower
+        self.upper = upper
+        self.body_fun = body_fun
+
+    def call(self, init_val):
+        return backend.core.fori_loop(
+            self.lower,
+            self.upper,
+            self.body_fun,
+            init_val,
+        )
+
+    def compute_output_spec(self, init_val):
+        return KerasTensor(init_val.shape, dtype=init_val.dtype)
+
+
+@keras_core_export("keras_core.ops.fori_loop")
+def fori_loop(lower, upper, body_fun, init_val):
+    """
+    For loop implementation.
```

Comment:
Move one-liner to the line above.

Reply:
I don't understand...
Do you mean like this?
```
""" For loop implementation.
```

or something else?